config/os.go

```diff
@@ -44,7 +44,6 @@ func GetEOL(family, release string) (eol EOL, found bool) {
 			"2":    {StandardSupportUntil: time.Date(2025, 6, 30, 23, 59, 59, 0, time.UTC)},
```

Comment:
```suggestion
			"2":    {StandardSupportUntil: time.Date(2026, 6, 30, 23, 59, 59, 0, time.UTC)},
```

cf. https://aws.amazon.com/jp/amazon-linux-2/faqs/